ChatSecure/Classes/View Controllers/OTRMessagesViewController+SupplementaryViews.swift

```diff
@@ -0,0 +1,252 @@
+//
+//  OTRMessagesViewController+NewDevice.swift
+//  ChatSecureCore
+//
+//  Created by N-Pex on 26/2/18.
+//  Copyright © 2018 Chris Ballinger. All rights reserved.
+//
+
+import Foundation
+import OTRAssets
+
+var OTRMessagesViewController_supplementaryViewsAssociatedObject: UInt8 = 0
+var OTRMessagesViewController_prototypeCellsAssociatedObject: UInt8 = 0
+
+extension OTRMessagesViewController: OTRMessagesCollectionViewFlowLayoutSupplementaryViewProtocol {
```

Comment:
Can you make this a standalone class conforming to OTRMessagesCollectionViewFlowLayoutSupplementaryViewProtocol and store it as a private member inside OTRMVC.m? You should be able to make it standalone if you just init it with DatabaseConnections and pass in the message at indexPath in supplementaryViewsForCellAtIndexPath. I'd prefer to avoid objc_getAssociatedObject :)